meteomatics/logger.py

```diff
@@ -11,4 +11,14 @@ def create_log_handler(fmt='%(asctime)s| %(levelname)s |%(message)s', stream=sys
     handler = logging.StreamHandler(stream=stream)
     handler.setFormatter(formatter)
 
-    logging.getLogger(LOGGERNAME).addHandler(handler)
\ No newline at end of file
+    logging.getLogger(LOGGERNAME).addHandler(handler)
+
+
+def set_log_level(level):
```

Comment:
What I meant was to specify the type of the `level` variable here... however, since it is going to be an int, there is no really nice way to define the type of `level` without adding more code... and even there, I am not sure it would be more precise. 

Maybe I would add something like

```suggestion
def set_log_level(level):  # E.g. set_log_level(logging.DEBUG)
```